src/run-graph.ts

```diff
@@ -81,6 +80,18 @@ export class RunGraph {
     else this.consoles = new DefaultConsole()
   }
 
+  private globalPrefixer = new Prefixer().prefixer
+  get prefixer() {
```

Comment:
maybe change to a method createOrProvidePrefixerForProcess to signify that invocation can now create a new one?